Moodis/Moodis/Ui/MenuMethods.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Moodis.Network.Face;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Moodis.Ui
+{
+    class MenuMethods
```

Comment:
Change the name of this class to MenuViewModel as it contains business logic for MenuView.